halo2_proofs/src/multicore.rs

```diff
@@ -2,8 +2,15 @@
 //! `halo2`. It's currently just a (very!) thin wrapper around [`rayon`] but may
```

Comment:
```suggestion
//! `halo2`. It's currently just a (very!) thin wrapper around `rayon` but may
```
(This link broke because `rayon` is no longer a direct dependency.)